opensrp-reveal/src/main/java/org/smartregister/reveal/server/JavaHTTPServer.java

```diff
@@ -0,0 +1,143 @@
+package org.smartregister.reveal.server;
+
+import org.apache.commons.lang3.StringUtils;
+import org.smartregister.reveal.BuildConfig;
+import org.smartregister.reveal.application.RevealApplication;
+import org.smartregister.util.Utils;
+
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Date;
+import java.util.StringTokenizer;
+
+public class JavaHTTPServer implements Runnable{
+
+    static final String DEFAULT_STYLE_JSON_FILE = "map-download-style.json";
+    static final String DEFAULT_DG_ID_PLACEHOLDER = "DIGITAL_GLOBE_ID";
+    public static String digitalGlobeIdPlaceHolder;
+    public static String styleJsonFile;
+    static String styleJson;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)